packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -70,6 +70,33 @@ class has additional `size` and `offset` properties:
 <ApiSnippet url={'/api/2d/components/Layout#offset'} />
 <hr />
 
+## Opting Out of Layout 
+
+You can disable the layout behaviour by passing `layout={false}` to your component:
+
+```tsx
+<Layout layout direction="row">
+  <Rect width={100} height={100} fill="green"/>
+  <Rect width={100} height={100} fill="red" layout={false} x={200} y={50}/>
+</Layout>
+```
+
+This behaves almost similar to **absolute postioning** in CSS Flexbox: 
```

Comment:
"almost similar" is redundant, and we don't frequently use bold like this. I'd recommend either making it a hyperlink to the MDN docs, or removing it. Also, positioning is misspelled.

```suggestion
This behaves like absolute positioning in CSS Flexbox: 
```